src/app/routes/home/getInitialData/index.js

```diff
@@ -39,9 +64,13 @@ export default async ({ path: pathname, service, pageType, toggles }) => {
       pageData: {
         ...transformJson(json),
         radioSchedulePosition,
+        ...(pageHasUsElectionsBanner
+          ? await fetchElectionsOembed(service)
+          : null),
       },
     };
   } catch ({ message, status = getErrorStatusCode() }) {
+    console.log(message);
```

Comment:
Is this to remain?

Reply:
ahhhh, thought I cleaned all of these up 👍

Reply:
done